deu/match-cleared.vbs

```diff
@@ -850,7 +850,9 @@ IF claim_referral_tracking_dropdown <> "Not Needed" THEN
     IF case_note_only = TRUE THEN CALL write_variable_in_case_note("Maxis case is inactive unable to add or update MISC panel")
     CALL write_variable_in_case_note("-----")
     CALL write_variable_in_case_note(worker_signature)
+	PF3 ' to ensure a new case note is started'
 END IF
+
```

Comment:
Should there be a PF3 within the IF...statement starting on line 923?

Reply:
they should be reviewing it but because it has been there for so long I think it is a good idea to keep it there